stdlib/messaging/kafka/src/main/ballerina/src/kafka/consumer.bal

```diff
@@ -119,8 +126,8 @@ public type ConsumerConfig record {|
 # + timestamp - Timestamp of the record, in milliseconds since epoch.
 # + topic - Topic to which the record belongs to.
 public type ConsumerRecord record {|
-    Data key;
-    Data value;
+    any key;
+    any value;
```

Comment:
Should this be `anydata` instead of `any`?

Reply:
It should be `any` since we are supporting custom serializers. Users can define their own `record` types (or `object` types), and define their own serializing, deserializing methods.